ibtk/src/solvers/impls/CCPoissonLevelRelaxationFACOperator.cpp

```diff
@@ -142,20 +142,12 @@ CCPoissonLevelRelaxationFACOperator::CCPoissonLevelRelaxationFACOperator(const s
           CELLG,
           input_db,
           default_options_prefix),
-      d_level_solver_type(),
-      d_level_solvers(),
-      d_level_solver_db(),
-      d_coarse_solver(nullptr),
-      d_coarse_solver_db()
+      d_level_solver_type(CCPoissonSolverManager::PETSC_LEVEL_SOLVER),
+      d_level_solver_default_options_prefix(default_options_prefix + "level_")
 {
     // Set some default values.
     d_prolongation_method = "LINEAR_REFINE";
```

Comment:
Header for these inits?

Reply:
These were members that were inherited from a parent class. Some of these override default values like
`d_prolongation_method = "LINEAR_REFINE";`
overrides the default value of
`"DEFAULT"` provided in `PoissonFACPreconditionerStrategy.h`
but others are the same and could be deleted such as
`d_coarse_solver_rel_residual_tol = 1.0e-5;`

I can go back and check all of these variables.